pdsClient/pdsClient.go

```diff
@@ -139,6 +139,17 @@ func (c *E3dbPDSClient) WriteRecord(ctx context.Context, params WriteRecordReque
 	return result, err
 }
 
+// DeleteRecord attempts to delete a record in e3db, returning error (if any).
+func (c *E3dbPDSClient) DeleteRecord(ctx context.Context, params DeleteRecordRequest) error {
```

Comment:
Please write an integration test for this. You have to run the pds and search services, and export some env variables for the integration tests to talk to those endpoints(all documented in the readme, but to save you some time
```
export E3DB_API_URL=https://local.e3db.com
export E3DB_API_KEY_ID=1d66c182779e47714cd957d4760a121580598d68e064c948e5887662c197538b
export E3DB_API_KEY_SECRET=aaadbd1cc72870e6bd41befa1273b4513d0d1fa04fb913257eec6158b6a35d75
export E3DB_CLIENT_ID=864e4b87-9eda-43fb-ae6d-d07d4275c73a
```